testsuite/tests/shadow_include/shadow_all.ml

```diff
@@ -100,11 +100,11 @@ end
 Line 4, characters 2-11:
 4 |   include S
       ^^^^^^^^^
-Error: Illegal shadowing of included type t/131 by t/146
+Error: Illegal shadowing of included type t/2 by t
        Line 2, characters 2-11:
-         Type t/131 came from this include
+         Type t/2 came from this include
        Line 3, characters 2-24:
-         The value ignore has no valid type if t/131 is shadowed
+         The value ignore has no valid type if t/2 is shadowed
```

Comment:
These could be sub-messages with their own source highlighting, like in https://github.com/ocaml/ocaml/pull/11679

Reply:
Good idea, but I think it is probably better to discuss this change of format once I will have submitted my second PR in this series which handle the case where there is a set of illegal includes but no single include is illegal in isolation.

Reply:
I couldn't resist to the temptation of switching to sub-messages since the implementation is straightforward and I find the resulting error message quite better.